README.md

```diff
@@ -391,6 +391,64 @@ _extends THREE.Camera_
 
 A class indicating that the path tracer should render an equirectangular view. Does not work with three.js raster rendering.
 
+## TemporalResolve
+
+A class that implements temporal filtering to preserve samples when the camera is moving. This helps reduce noise on camera movement by reprojecting the last frame's samples into the current one.
+
+### .temporalResolveMix
+
+```javascript
+temporalResolveMix = 0.9 : Number
+```
+
+How much the last frame should be blended into the current one. Higher values will result in a less noisy look at the cost of more smearing.
+
+### .clampRadius
+
+```javascript
+clampRadius = 1 : Number
+```
+
+An integer to set the radius of pixels to be used for neighborhood clamping. Higher values will result in a less noisy look at the cost of more blurring.
```

Comment:
What does "neighborhood clamping" mean here?

Reply:
You clamp the color of the reprojected pixel by the min and max color of the neighboring pixels from the raytracer's output at the current pixel. This is the most popular method to reduce ghosting for TRAA but since we have a noisy output in the first frames when raytracing, neighborhood clamping will lead to flickering there. So by increasing the number of pixel we use for neighborhood clamping, we reduce flickering but will have more smear.

Reply:
Thanks for the explanation! Maybe we can include more information on the docs about the role color plays in this process

Reply:
Sure, yeah I'll add a more in-depth explanation for it